Documentation/rkt-vs-other-projects.md

```diff
@@ -0,0 +1,122 @@
+# rkt vs other projects
+
+This document describes how rkt compares to various other projects in the container ecosystem.
+
+* [rkt vs Docker](#rkt-vs-docker)
+  * [Process Model](#process-model)
+  * [Privilege Separation](#privilege-separation)
+* [rkt vs runC](#rkt-vs-runc)
+* [rkt vs LXC/LXD](#rkt-vs-lxclxd)
+* [rkt vs OpenVZ](#rkt-vs-openvz)
+* [rkt vs systemd-nspawn](#rkt-vs-systemd-nspawn)
+* [rkt vs machinectl](#rkt-vs-machinectl)
+* [rkt vs qemu-kvm, lkvm](#rkt-vs-qemu-kvm-lkvm)
+
+## rkt vs Docker
+
+The Docker Engine, or simply Docker, is an application container runtime system that runs as a centralized API daemon. Docker can take `a "Docker Image" name, such as `quay.io/coreos/etcd`, and download, execute, and monitor the application.
```

Comment:
Should "Docker Image" link to the [Docker Image Specification](https://github.com/docker/docker/blob/master/image/spec/v1.md)?